MetaMask/metamask-extension
# fix: relax network symbol length validation

## **Description**

When adding a network with `wallet_addEthereumChain`, MetaMask was requiring the chain's symbol to be minimum of 2 characters.  But there are networks with single character symbols.  And MetaMask already lets you *manually* add a network with a single character symbol.  And it renders fine in the UI.  So relaxing the validation.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24872?quickstart=1)

## **Related issues**


## **Manual testing steps**

1. Visit https://chainid.network/
2. Connect wallet
3. Search for chainid `35443`
4. Add chain
5. Verify it added successfully

## **Screenshots/Recordings**



### **Before**

<img width="628" alt="Screenshot 2024-05-29 at 8 10 39 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/6412a9ca-0570-46f9-9b4e-ccffdb4c6b51">


### **After**

<img width="614" alt="Screenshot 2024-05-29 at 8 11 25 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/503192c9-05e1-4a88-9ed8-25d95c45afe0">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
